inc/environment_indicator/namespace.php

```diff
@@ -88,6 +89,32 @@ function add_admin_bar_dashboard_link( WP_Admin_Bar $wp_admin_bar ) {
 	] );
 }
 
+/**
+ * Add menu item to the Altis logo in the admin bar for creating a support ticket.
+ *
+ * @param WP_Admin_Bar $wp_admin_bar WP_Admin_Bar instance, passed by reference.
+ */
+function add_admin_bar_support_ticket_link( WP_Admin_Bar $wp_admin_bar ) {
+	if ( ! is_admin_bar_showing() ) {
+		return;
+	}
+
+	$env_name    = get_environment_name();
+	$support_url = 'https://dashboard.altis-dxp.com/#/support/new';
```

Comment:
This should be a `const` in the namespace rather than a variable here

Reply:
Think it's helpful to explain why you'd do that as well